docs/markdown/Installing.md

```diff
@@ -115,7 +115,7 @@ running `sudo meson install` will drop permissions and rebuild out of date
 targets as the original user, not as root.
 
 *(since 1.1.0)* Re-invoking as root will try to guess the user's preferred method for
-re-running commands as root. The order of precedence is: sudo, doas, pkexec
+re-running commands as root. The order of precedence is: run0, sudo, doas, pkexec
```

Comment:
I would consider it unacceptable to change the order to stop using sudo as the default.

Reply:
I changed the ordering.

Reply:
Per standard operating procedure, please change the ordering using `git commit --amend && git push --force` to commit the results, not a second commit. :)